cms/templates/js/upload-dialog.underscore

```diff
@@ -1,13 +1,13 @@
-<form class="upload-dialog" method="POST" action="<%= url %>" enctype="multipart/form-data">
-        <p id="dialog-assetupload-description" class="message"><%= message %></p>
+<form class="upload-dialog" method="POST" action="<%- url %>" enctype="multipart/form-data">
+        <p id="dialog-assetupload-description" class="message"><%- message %></p>
```

Comment:
FYI: For messages like this one and `error.message` below, just double check that some isn't using the template to sometimes display HTML messages.  If that is sometimes the case, this would need to be ``<%= HtmlUtils.ensureHtml(message) %>`, and `message` would need to be created using HtmlUtils when it wants to be an HTML message.  That is probably not the case here, but just making you aware.  Thanks!